src/chrome/content/rules/AllShare_Play.xml

```diff
@@ -1,33 +1,26 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://allshareplay.com/ => https://www.allshareplay.com/: (35, 'Unknown SSL protocol error in connection to www.allshareplay.com:443 ')
-	For other Samsung coverage, see Samsung.xml.
-
-
-	Problematic subdomains:
-
-		- ^	(cert only matches *.allshareplay.com)
-
-
-	Fully covered subdomains:
-
-		- (www.)	(^ → www)
-		- m
-
+	For other Samsung coverage, see Samsung.com.xml.
+	
+	Mismatch:
+	sw.*
 -->
-<ruleset name="AllShare Play" default_off='failed ruleset test'>
-
-	<target host="allshareplay.com" />
-	<target host="*.allshareplay.com" />
-
-
-	<securecookie host="^.*\.allshareplay\.com$" name=".+" />
 
+<ruleset name="AllShare_Play">
 
-	<rule from="^http://(?:www\.)?allshareplay\.com/"
-		to="https://www.allshareplay.com/" />
-
-	<rule from="^http://(fwk|m)\.allshareplay\.com/"
-		to="https://$1.allshareplay.com/" />
-
+	<target host="allshareplay.com" />
+	<target host="www.allshareplay.com" />
+	<target host="fwk.allshareplay.com" />
+	<target host="m.allshareplay.com" />
+		<!--	403 in http://m.allshareplay.com/	-->
+		<exclusion pattern="^http://m\.allshareplay\.com/$" />
+			<test url="http://m.allshareplay.com/" />
+
+	<rule from="^http://(?:www\.)?allshareplay\.com/$"
```

Comment:
This is only needed for ^allshareplay.com/$, www works server side https. Please also document why this is needed (403).